lib/pages/chat/events/message/message.dart

```diff
@@ -120,6 +123,12 @@ class Message extends StatelessWidget {
             MessageTypes.Audio,
           }.contains(event.messageType);
 
+          final hideDisplayNameInBubbleChat = {
+            MessageTypes.Video,
+            MessageTypes.Image,
+            MessageTypes.File,
```

Comment:
Please demo case for files, because the timeline for file is like message text

Reply:
done